lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/JWTVerifier.java

```diff
@@ -21,22 +22,31 @@
 import java.security.interfaces.RSAPublicKey;
 import java.text.ParseException;
 import java.util.Base64;
-import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class JWTVerifier {
 
     public void verifyJWT(
             Map<String, String> clientAuthenticationConfig,
             SignedJWT signedJWT,
-            List<String> requiredClaims)
+            Set<String> requiredClaims)
             throws SessionValidationException, ClientConfigurationException {
         this.verifyJWTHeader(clientAuthenticationConfig, signedJWT);
         this.verifyJWTClaimsSet(clientAuthenticationConfig, signedJWT, requiredClaims);
         this.verifyJWTSignature(clientAuthenticationConfig, signedJWT);
     }
 
+    public void verifyJWT(Map<String, String> clientAuthenticationConfig, SignedJWT signedJWT)
+            throws SessionValidationException, ClientConfigurationException {
+        this.verifyJWTHeader(clientAuthenticationConfig, signedJWT);
+        this.verifyJWTClaimsSet(
+                clientAuthenticationConfig,
+                signedJWT,
+                Set.of(JWTClaimNames.EXPIRATION_TIME, JWTClaimNames.SUBJECT));
+        this.verifyJWTSignature(clientAuthenticationConfig, signedJWT);
```

Comment:
You can reuse the first method call something like
```
        verifyJWT(
                clientAuthenticationConfig,
                signedJWT,
                Set.of(JWTClaimNames.EXPIRATION_TIME, JWTClaimNames.SUBJECT));
```

Reply:
Absolutely!